migrations/etl/1_Mine_profile_detail_location.sql

```diff
@@ -147,6 +150,43 @@ BEGIN
         (new.lat_dec <> 0 AND new.lon_dec <> 0);
     SELECT count(*) FROM mine into new_row;
     SELECT count(*) FROM mine_location into location_row;
+
+    -- Upsert data from new_record into mine_type
+    WITH new_record AS (
+        SELECT *
+        FROM ETL_PROFILE
+        WHERE NOT EXISTS (
+            SELECT  1
+            FROM    mine_type
+            WHERE   mine_no = ETL_PROFILE.mine_no
+        )
+    )
+    INSERT INTO mine_type(
+        mine_type_guid       ,
+        mine_guid            ,
+        mine_tenure_type_code,
+        create_user          ,
+        create_timestamp     ,
+        update_user          ,
+        update_timestamp     )
+    SELECT
+        gen_random_uuid()   ,
+        new.mine_guid       ,
+        CASE
+          when new.mine_typ = ANY('{CX,CS,CU}'::text[]) THEN 'COL'
```

Comment:
Looks like we're collapsing types here. Where did we get this mapping?

Reply:
From business (Aaron and Brian)